google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.java

```diff
@@ -83,6 +74,13 @@ public void actionPerformed(ActionEvent event) {
     });
 
     setAnchor(myWebArtifactToDeployLabel);
+    appEngineSettingsPanel.setBorder(PlainSmallWithoutIndent.createTitledBorder(
```

Comment:
What does this do? I couldn't make out any noticeable changes from your screenshot.